src/lib/components/forms/picture-input.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+  import Icon from '$lib/components/icons/icon.svelte';
+  import Icons from '$lib/components/icons/icons';
+
+  export let text: string;
+  let avatar, fileinput;
+
+  const onFileSelected = (e) => {
+    let image = e.target.files[0];
+    let reader = new FileReader();
+    reader.readAsDataURL(image);
+    reader.onload = (e) => {
+      avatar = e.target.result;
```

Comment:
```suggestion
  if (e.target?.result) {
        avatar = e.target.result.toString();
      }
```

My IDE nagged me about this because e.target might be null in error cases ;p